bindings/python/tools/python/pyproject.toml.in

```diff
@@ -3,3 +3,6 @@
 [build-system]
 requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
+
```

Comment:
I added a pyproject.toml file that can actually be recognized by the black formatter so if you have format on save enabled in vscode it will choose the proper line length.
Along with this, the black --line-length=90 should be removed from the ostk-format-python helper in ostk-base (in this MR https://github.com/open-space-collective/open-space-toolkit/pull/80) since each ostk repo should replace this with a pyproject file in the root of the bindings.

The second pyproject.toml.in file is a template file that's meant for the actual package setup/configuration needed by CMake. I just added the black configuration settings to that one too cause it can't hurt.